### ISSUE TITLE
Performance Issues when using Grand 

### ISSUE THREAD

**Issue by chilicrabcakes** [aplbrain/grand-cypher#54]
Hey guys - great library first of all.

I'm creating a graph using the grand library and saving as an SQLite backend. I want to query the graph using Cypher, so I installed GrandCypher and am running the code like this:
```
import grand
from grandcypher import GrandCypher

G = grand.Graph(
    backend=grand.backends.SQLBackend(
        db_url="sqlite:///my_graph.db",
        directed=False
    )
)
```

Then I'm creating a GrandCypher object like this:
```
gc_graph = GrandCypher(G.nx)
```

However, this step is taking more than half an hour to complete. My graph has 13090 nodes and 19253 edges. Is this expected?

I'm able to run networkx functions normally, like G.nx.nodes() and G.nx.edges() for example.

**Comment by chilicrabcakes** [comment]
this may be linked to https://github.com/aplbrain/grand-cypher/issues/52

**Comment by j6k4m8** [comment]
Almost certainly you're right that this is delay caused by [multidigraph conversion](https://github.com/aplbrain/grand-cypher/blob/af754de6e65a03dd4c0d3a6e084e175f5c26b756/grandcypher/__init__.py#L402)

So I guess that's a trick question — _yes_ that's expected right now, but _definitely_ broken behavior and bad and we must fix it :) 

While we work on that, it's possible that commenting out that line that checks and converts may be a good short-term solution... Also this graph may be of a reasonable size to simply run in-memory rather than with the SQL backend... Would be happy to chat more about this to make sure we have a working solution for your needs right away!!

**Comment by davidmezzetti** [comment]
Running `pip install grand-cypher==0.9.0` will work around the issue for now too.

**Comment by j6k4m8** [comment]
Wow thank you @davidmezzetti for jumping in with a workaround, and thank you @jackboyla for the great fix in #55! This fix is now live on PyPI and can be installed with `pip install -U grand-cypher` or by otherwise installing at least version 0.11.0.

Amazing team!

@chilicrabcakes give it a whirl and see if this fixes your performance issue?

**Comment by chilicrabcakes** [comment]
Awesome! I'll try the new fix, let you all know how it goes

**Comment by davidmezzetti** [comment]
Seems like this issue is probably addressed and can be closed?

**Comment by j6k4m8** [comment]
Yes that's right! @chilicrabcakes feel free to reopen if you are still experiencing issues :)
